examples/browser/package.json

```diff
@@ -17,25 +17,24 @@
     }
   },
   "dependencies": {
-    "@theia/core": "latest",
-    "@theia/filesystem": "latest",
-    "@theia/workspace": "latest",
-    "@theia/preferences": "latest",
-    "@theia/navigator": "latest",
-    "@theia/process": "latest",
-    "@theia/terminal": "latest",
-    "@theia/editor": "latest",
-    "@theia/markers": "latest",
-    "@theia/monaco": "latest",
-    "@theia/messages": "latest",
-    "@theia/cpp-debug": "latest",
-    "@theia/vsx-registry": "latest",
-    "@theia/keymaps": "latest",
-    "@theia/getting-started": "latest",
+    "@theia/core": "1.17.0",
+    "@theia/filesystem": "1.17.0",
+    "@theia/workspace": "1.17.0",
+    "@theia/preferences": "1.17.0",
+    "@theia/navigator": "1.17.0",
+    "@theia/process": "1.17.0",
+    "@theia/terminal": "1.17.0",
+    "@theia/editor": "1.17.0",
+    "@theia/markers": "1.17.0",
+    "@theia/monaco": "1.17.0",
+    "@theia/messages": "1.17.0",
+    "@theia/vsx-registry": "1.17.0",
+    "@theia/keymaps": "1.17.0",
+    "@theia/getting-started": "1.17.0",
```

Comment:
On these lines and related lines, it may be preferable to keep the package.json entries set to `latest` and instead update only the lockfile entries to 1.17.0.

Reply:
Then everytime we run a `yarn` upgrade we get bitten by a Theia upgrade as well... I'm more in favor of explicitly targeting what we plan on using. Which reminds me that I should update this to target 1.17.2 now.

Reply:
there's also the issue that Theia's versioning can be quite problematic as it follows romantic version rather than semantic. So if we target `latest` but only build against 1.17.2 in this repository because of the lockfile, then the day someone pulls that Trace extension when 1.18.0 is released with breaking changes there's no telling if the extension will keep working...

Reply:
I'm ok to be more explicit about the Theia version that the theia-trace-extension is developed against. In the end, when we upgrade to new Theia version, we need to update the version numbers and do yarn upgrade. So, not much more work.

Reply:
Especially if you prefer to do an upgrade of everything but Theia first, and then upgrade Theia like you asked me to do for this PR. If you target `latest` then doing `yarn upgrade` will do everything at once.